boards/arm/nrf5340dk_nrf5340/CMakeLists.txt

```diff
@@ -46,13 +46,10 @@ if (CONFIG_BUILD_WITH_TFM)
 		# Merge tfm_s and zephyr (NS) image to a single binary.
 		set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
 			COMMAND ${SREC_CAT}
-			ARGS ${CMAKE_BINARY_DIR}/tfm/bin/tfm_s.bin -Binary
+			ARGS $<TARGET_PROPERTY:tfm,TFM_S_BIN_FILE> -Binary
```

Comment:
Nothing wrong here, comment relates to this commit message: f394554937510bd90300eabf98f88f8a24bdd217
Please provide reference to upstream PR.

Reply:
Fixed